doc/development_process/proposals.rst

```diff
@@ -0,0 +1,24 @@
+
+Proposals and RFCs
+##################
+
+Feature or enhancement proposals for Zephyr are used to propose a new design or
+modify an existing feature and get it approved. To better integrate with the
+current tools used in the project and to enable traceability and archiving of
+such proposals a unified format for enhancement proposals is used.
+
+Procedures
+***********
+
+Maintain such proposals in a document format (similar to the documentation
```

Comment:
Not sure about this. I think we should keep RFCs in either GitHub issues or Pull Requests, not documents?